doc/content/sfr/abtr_xsgen_workflow/abtr_xsgen_workflow.md

```diff
@@ -17,6 +17,7 @@
                        computing_needs:HPC
                        cross_sections:MCC3
                        fiscal_year:2024
+                       sponsor:NEAMS
```

Comment:
adding this one already in my PR

I think you may want to add "cross section generation" to the input_features?

Reply:
Reverted last commit and added cross_section_generation to input_features